example_site/src/components/TemplateBrowser.js

```diff
@@ -16,8 +16,15 @@ function Template({ template }) {
   return (
     <>
       <button className="template-button" title={template.id} onClick={() => openEditor({ template })}>
+        <div className='template-container'>
           <HoverVideoPlayer
-            className="template-container"
+            // className="video-container"
+            style={{
+              width: template.width > template.height ? '200px' :
+                (template.width/(template.height/200)),
+              height: template.height > template.width ? '200px' : 
+                (template.height/(template.width/200)),
+            }}
```

Comment:
I think this is cutting off some of the tall thumbnail images (but not the hover preview) let's take a look during our 1:1